ui/admin/tests/e2e/tests/auth-method-ldap.spec.js

```diff
@@ -0,0 +1,141 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+/* eslint-disable no-undef */
+const { test, expect } = require('@playwright/test');
+const { execSync } = require('child_process');
+const { checkEnv, authenticatedState } = require('../helpers/general');
+const { nanoid } = require('nanoid');
+
+const {
+  authenticateBoundaryCli,
+  checkBoundaryCli,
+  deleteOrgCli,
+} = require('../helpers/boundary-cli');
+const {
+  createNewOrg,
+  createRole,
+  addPrincipalToRole,
+} = require('../helpers/boundary-ui');
+
+test.use({ storageState: authenticatedState });
+
+test.beforeAll(async () => {
+  await checkEnv([
+    'E2E_LDAP_ADDR',
+    'E2E_LDAP_DOMAIN_DN',
+    'E2E_LDAP_ADMIN_DN',
+    'E2E_LDAP_ADMIN_PASSWORD',
+    'E2E_LDAP_USER_NAME',
+    'E2E_LDAP_USER_PASSWORD',
+    'E2E_LDAP_GROUP_NAME',
+  ]);
+
+  await checkBoundaryCli();
+});
+
+test('Set up LDAP auth method @ce @ent @docker', async ({ page }) => {
+  await page.goto('/');
+  let org;
+  let connect;
```

Comment:
not a blocker- I think these two can be `const`, I don't see them being reassigned anywhere

Reply:
I believe those need to be `let`. If I change to `const`, I get this warning
```
'const' declarations must be initialized.ts (1155)
```